features/unit/dryrun_trace.feature

```diff
@@ -0,0 +1,13 @@
+@unit
+Feature: Dryrun Trace
+
+    @unit.dryrun.trace.application
+    Scenario Outline: Application Trace Tests
+        Given a dryrun response file "<dryrun-response-file>" and a transaction id "<txn-id>"
+        When I call app trace
+        Then the output should equal "<golden>"
```

Comment:
```suggestion
        Then calling app trace produces "<app-trace-file>"
```
I'm recommending this step consolidation because `the output should equal` step seems too generic. I would guess something like that already exists, in which case you can re-use that step.

Also I'm recommending renaming `golden` to `app-trace-file` because `golden` usually refers to an inline string, while `*-file` is more clearly referring to a file.